src/modules/template-details/TemplateDetails.tsx

```diff
@@ -82,6 +96,16 @@ const TemplateDetails = ({ match }: TemplateDetails.Props) => {
                 </Button>
               </a>
 
+              <Guard.Protected>
+                {({ user }) =>
+                  user.connectedWithGithub && (
+                    <Button onClick={openFork}>
```

Comment:
On button double click - text in dialog is automatically selected. First thing user should see is error or success font color, but when text is selected on dialog box opening user would have either unselect it first in order to see result color or read the whole message. In both cases it's not the best user experience.